<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1819</width>
    <height>982</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>910</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="performButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>910</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Выполнить</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>1220</x>
     <y>10</y>
     <width>591</width>
     <height>741</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Номер кольца</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>С столба</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>На столб</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="refreshButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>910</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Заново</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
